cpp/src/barretenberg/honk/composer/ultra_honk_composer.test.cpp

```diff
@@ -41,12 +41,13 @@ std::vector<uint32_t> add_variables(auto& composer, std::vector<fr> variables)
  */
 void verify_consistency(honk::UltraProver& honk_prover, plonk::UltraProver& plonk_prover)
 {
-    // Check that all lagrange polys agree
+    // Check that all lagrange polys agree (except for sigmas/ids which are constructed differently for Honk)
```

Comment:
I'm keeping these plonk-honk consistency checks around for now but they're slowly becoming less relevant (and will eventually be replaced with the normal composer tests). Here for example I've removed the consistency check on the permutation polys since we construct them differently for plonk and honk.